### PR TITLE
Feature/tao 7267 remove jqueryui

### PR DESCRIPTION
**Pull request by krampstudio**
Linked to https://oat-sa.atlassian.net/browse/TAO-7267 

What has changed : 
 - remove jquery-ui from the permission pages
 - It was used to create "tabs" to handle the show/hide of the Users/Roles tables if they had too much entry.
 - I've replaced this hidden feature by : 
 1. optimize the space by using the correct heading level for the page title, on all the other pages the page title is `h2` not `h1`.
 2. make the table scrollable if it becomes to long

How to test : 
 - Open the "Access Control" menu on an Item Class

image

Dependent PR : 
 - [x] https://github.com/oat-sa/tao-core/pull/2076

### CODE REVIEW

File: views/templates/AdminAccessController/index.tpl [hunk 1]
```diff
@@ -1,16 +1,12 @@
 <?php use oat\tao\helpers\Template;?>
 <div class="permission-container flex-container-full">
-    <h1><?= __('Access Permissions for') ?> <em><?= get_data('label') ?></em></h1>
+    <h2><?= __('Access Permissions for') ?> <em><?= get_data('label') ?></em></h2>
```
Comment (ferenckiss89) [inline views/templates/AdminAccessController/index.tpl]: What is the reason behind this change?

**Comment by ferenckiss89** [comment]
Please provide a description to the pr

**Comment by krampstudio** [comment]
@ferenckiss89 my bad I should have created the pr in draft mode.

File: views/templates/AdminAccessController/index.tpl (hunk 1 above)
Comment (krampstudio) [reply views/templates/AdminAccessController/index.tpl]: Getting space and using the same style than everywhere else by the way...
